packages/inter-protocol/test/auction/test-auctionContract.js

```diff
@@ -1290,3 +1291,44 @@ test('bid zero', async t => {
     },
   );
 });
+
+test('time jumps forward', async t => {
+  const { collateral, currency } = t.context;
+  const driver = await makeAuctionDriver(t);
+
+  const liqSeat = await driver.setupCollateralAuction(
+    collateral,
+    collateral.make(1000n),
+  );
+  await driver.updatePriceAuthority(
+    makeRatioFromAmounts(currency.make(11n), collateral.make(10n)),
+  );
+
+  const result = await E(liqSeat).getOfferResult();
+  t.is(result, 'deposited');
+
+  await driver.advanceTo(167n);
+  const seat = await driver.bidForCollateralSeat(
+    currency.make(200n),
+    collateral.make(200n),
+  );
+  t.is(await E(seat).getOfferResult(), 'Your bid has been accepted');
+  t.false(await E(seat).hasExited());
+
+  await driver.advanceTo(170n, 'wait');
+
+  // jump past end of auction
+  await driver.advanceTo(1500n, 'wait');
+
+  t.false(await E(seat).hasExited());
+  await E(seat).tryExit();
+  t.true(await E(seat).hasExited());
+
+  await assertPayouts(t, seat, currency, collateral, 200n, 0n);
+  await assertPayouts(t, liqSeat, currency, collateral, 0n, 1000n);
```

Comment:
please use consts defined at the top of the function so it's obvious that these are the same values that went into the bid